README.md

```diff
@@ -139,32 +138,40 @@ Values in keys:
 ```
 
 ## String IDs
-Always return string ids, some languages like JavaScript don't support big ints. Serialize/deserialize ints to strings if storing ids as ints.
+Always return string ids. Some languages, like JavaScript, don't support big ints. Serialize/deserialize ints to strings if storing ids as ints.
 
 ## Error handling
-Error responses should include a message for the user, internal error type (corresponding to some specific internally determined constant, which must be represented as a string), links where developers can find more info.
+Error responses should include a message for the user, an internal error type (corresponding to some
+specific internally determined constant represented as a string), and links to info for developers.
 
-There must only be one top level error. Errors should be returned in turn. This makes internal error logic simpler. It also makes it easier for API consumers to deal with errors.
+There must only be one top level error. Errors should be returned in turn. This makes internal error
+logic and dealing with errors as a consumer of the API easier.
```

Comment:
this is emacs line wrapping. I don't know if it'll produce ugly markdown output. If it does I'll fix it in a follow up.